spec/features/user/user_sign_in_spec.rb

```diff
@@ -0,0 +1,45 @@
+feature 'Users' do
+
+  context 'when signing up' do
+
+     scenario 'can sign up' do
+      user = build(:user)
+      expect{sign_up(user)}.to change(User, :count).by (1)
+      expect(page).to have_content("Welcome, #{user.username}")
+      expect(User.first.email).to eq("#{user.email}")
+    end
```

Comment:
Lint/BlockAlignment: `end` at 10, 4 is not aligned with `scenario 'can sign up' do` at 5, 5